gdal/frmts/grib/gribdataset.h

```diff
@@ -109,6 +109,7 @@ class GRIBDataset final: public GDALPamDataset
     GIntBig nCachedBytes;
     GIntBig nCachedBytesThreshold;
     int bCacheOnlyOneBand;
+    int nSplitAndSwap;
```

Comment:
A comment explaining the semantics of this variable, and perhaps renaming it to nsplitAndSwapColumn, would be welcome